src/image/loading_displaying.js

```diff
@@ -238,18 +238,10 @@ p5.prototype.saveGif = async function(
   if (typeof duration !== 'number') {
     throw TypeError('Duration parameter must be a number');
   }
-  // if arguments in the options object are not correct, cancel operation
-  if (typeof options.delay !== 'number') {
```

Comment:
How do you feel about putting these two checks back in, but checking `delay` and `units` rather than `options.delay` and `options.units`? That way, if one accidentally specifies units as, for example, 's' instead of 'seconds', they'll still hit an error. It will mean moving it to after we define those variables, I believe.

Reply:
Yeah, that should be good! I'm on it.

Reply:
```js
const delay = options?.delay ?? 0
const units = options?.units ?? 'seconds';
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
😉